scripts/make.py

```diff
@@ -50,6 +50,11 @@
 build_flags.define("USB_PRODUCT_ID", kb_json["usb"]["pid"])
 
 # Analog Configuration
+if "resolution" in kb_json["analog"]:
+    build_flags.define("ADC_RESOLUTION", kb_json["analog"]["resolution"])
+else:
+    build_flags.define("MAX_ADC_RESOLUTION", driver_json["metadata"]["adc_bits"])
```

Comment:
Since we will end up with ADC_RESOLUTION = MAX_ADC_RESOLUTION anyway, I think we can just define ADC_RESOLUTION here, and maybe change the "adc_bits" to "max_adc_bits" in driver.json for clearer meaning of this field.